seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2388,8 +2388,8 @@
         "assert(myTest(10) === \"10 or Over\", 'message: <code>myTest(10)</code> should return \"10 or Over\"');",
         "assert(myTest(11) === \"10 or Over\", 'message: <code>myTest(10)</code> should return \"10 or Over\"');",
```

Comment:
I know this is not your issue, but this one should say 11.

Could you fix it too, while you're at it?